deps/zlib/zlib.gyp

```diff
@@ -18,6 +18,7 @@
             'adler32.c',
             'compress.c',
             'contrib/optimizations/insert_string.h',
+            'cpu_features.c',
```

Comment:
Since we are adding `cpu_features.c` to `sources` here already, is there any need to re-add it to the `sources` below? I guess we could treat the `cpu_features.h` file too in the same way.

Reply:
I think that makes sense. Look afb64b0f1d1af49eddae884472d3d358223f10e4